docs/remote-dev.md

```diff
@@ -0,0 +1,68 @@
+# Visual Studio Code Remote Development
+
+[VSCode Remote Development](https://code.visualstudio.com/docs/remote/remote-overview) allows you to run your code and environment in a container. This is especially useful for Swift when developing on macOS and deploying to Linux to ensure there are no compatibility issues in Foundation when running your code.
+
+## Requirements
+
+As well as installing this extension, you must install Docker on your machine to run the remote container in. See the [Visual Studio Code documentation](https://code.visualstudio.com/docs/remote/containers) for more details.
+
+Next, install the [Remote Development extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) that contains extensions for working in remote environments in VSCode.
```

Comment:
Do we need to install the full remote development extension pack. You can just install remote containers https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers

Reply:
Should these two paragraphs be the other away around. You talk about installing the remote development extension in your paragraph about Docker as if it has already been done and then mention you should install it in the next paragraph.

Reply:
It was about installing the Swift extension, have clarified